### ISSUE TITLE
Configuration of MVC project with EF6

### ISSUE THREAD

**Issue by januszj** [thepirat000/Audit.NET#1]
Hi,
I'm testing your library for the auditing in my application that uses EF6.
I configured SQL provider in Application_Start.

With first of the AuditScope use I got this exception:
The default DbConfiguration instance was used by the Entity Framework before the 'DbConfig' type was discovered. An instance of 'DbConfig' must be set at application start before using any Entity Framework features or must be registered in the application's config file. See http://go.microsoft.com/fwlink/?LinkId=260883 for more information.

Can you explain how to use it in MVC app with EF6 in use and the same database as target ?

**Comment by thepirat000** [comment]
This is a known issue, and it's partly because EF6 only allows one DbConfiguration per app-domain. So if you have a different one, or if other DbContext was created before the Audit.SqlServer context, you will see this error.

As a workaround you can do the following in your application startup:
`DbConfiguration.SetConfiguration(new DbConfig());`

But I hope to fix the problem in a better way soon (will try removing the [DbConfigurationType(typeof(DbConfig))] on [AuditModel.Context.cs](https://github.com/thepirat000/Audit.NET/blob/master/Audit.Sql/AuditModel.Context.cs#L16)).

Thanks

**Comment by januszj** [comment]
Your workaround worked. But when I have Azure deployment I have following config:
```
public class MyConfiguration : DbConfiguration
    {
        public MyConfiguration()
        {
            SetExecutionStrategy("System.Data.SqlClient", () => new SqlAzureExecutionStrategy());
        }
    }
```
I understand this will not work with the workaround ?
Thanks

**Comment by thepirat000** [comment]
Actually the workaround set the class DbConfig as the default dbconfiguration, and that class uses the same strategy (SqlAzureExecutionStrategy)

https://github.com/thepirat000/Audit.NET/blob/master/Audit.Sql/DbConfig.cs

**Comment by thepirat000** [comment]
Closing this. Fixed on v2.6.

The workaround is not needed anymore
